configure.ac

```diff
@@ -3219,9 +3219,8 @@ dnl ***  LLVM  ***
 dnl **************
 
 AC_ARG_ENABLE(llvm,[  --enable-llvm	Enable the LLVM back-end], enable_llvm=$enableval, enable_llvm=default)
-AC_ARG_ENABLE(loadedllvm,[  --enable-loadedllvm	Load the LLVM back-end dynamically], enable_llvm=$enableval && enable_loadedllvm=$enableval, enable_loadedllvm=no)
```

Comment:
autoconf will just issue a warning if someone passes this now unknown option right?

Reply:
We still have --enable-loadedllvm in packaging/MacSDK/mono.py and README.md